src/ansys/units/dimensions.py

```diff
@@ -7,96 +9,99 @@
 class Dimensions:
     """
     A composite dimension (or simply dimensions) composed from an arbitrary number of
-    fundamental dimensions, where each fundamental dimension is a pair consisting of
-    base dimension and exponent.
+    dimensions, where each dimension is a pair consisting of base dimension and
+    exponent.
 
     A dictionary of ``BaseDimensions`` and exponent is required
     for a non-dimensionless object.
 
     Parameters
     ----------
-    dimensions_container : dict, optional
+    dimensions_dict : dict, optional
         Dictionary of {``BaseDimensions``: exponent, ...}.
-
-    Attributes
-    ----------
-    dimensions
     """
 
     def __init__(
         self,
-        dimensions_container: dict[ansunits.BaseDimensions, Union[int, float]] = None,
+        dimensions_dict: dict[ansunits.BaseDimensions, Union[int, float]] = None,
```

Comment:
```suggestion
        dimensions: dict[ansunits.BaseDimensions, Union[int, float]] = None,
```
Is it worth the suffix now that I can just read the type hint and see what it is?